deployment/sw360/docker-entrypoint.sh

```diff
@@ -84,6 +84,34 @@ if [ "$HTTPS_HOSTS" ]; then
 fi
 
 ################################################################################
+# Setup for trusted Certificate Authorities
+#
+# $TRUSTED_CACERTS should be a comma separated list of environment variable
+# names, e.g.:
+# "TRUSTED_CA1,TRUSTED_CA2"
+# Mentioned variables ($TRUSTED_CA1 and $TRUSTED_CA2 in this example) should
+# contain the certificates, e.g.:
+# -----BEGIN CERTIFICATE-----
+# AAAAAAAAAAAAAAAAAAAAAAAAAAAA
+# -----END CERTIFICATE-----
+if [ "$TRUSTED_CACERTS" ]; then
+    for CERT_NAME in $(echo $TRUSTED_CACERTS | sed "s/,/ /g"); do
+        if [ "${!CERT_NAME}" ]; then
+          echo "Trust certificate $CERT_NAME ..."
+          while read -r line; do
+            echo $line >> $CERT_NAME.crt
```

Comment:
It might be best practice to save the temporary files in an temporary place, e.g. use something roughly like
```
    OUT="$(mktemp).crt"
    echo -e $CERT_NAME" > $OUT
    keytool [...] -import -file $OUT
```